topdown/crypto.go

```diff
@@ -410,6 +443,9 @@ func getRSAPrivateKeyFromPEM(pemBlocks []byte) (interface{}, error) {
 	if p.Type == blockTypePrivateKey {
```

Comment:
suggestion: I think we can remove the comments `// if the key is in PKCS8 format` etc for consistency.

We might also want to make this a switch statement now there are more than 2 options too.